src/NEO/Signer.h

```diff
@@ -0,0 +1,37 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "../PrivateKey.h"
+#include "../Data.h"
+#include "Address.h"
+#include "Transaction.h"
+
+namespace TW::NEO {
+
+class Signer {
+  private:
+    Data publicKey;
+    TW::PrivateKey privateKey;
+    Address* address;
```

Comment:
Why is a pointer needed here?  Why couldn't it be value type?

Reply:
I didn’t want to add a default constructor with an initialization like Address(«1111111111111111111114oLvT2»). I start extend address class for this and remove pointer